bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -100,6 +103,18 @@ def add_or_update_advisory(data, html):
     return advisory
 
 
+def add_hofers(filename, data):
+    program = os.path.basename(filename)[:-4]
+    HallOfFamer.objects.filter(program=program).delete()
+    for hofer in data['names']:
+        HallOfFamer.objects.create(
+            program=program,
+            name=hofer['name'],
+            date=hofer['date'],
+            url=hofer.get('url', ''),
```

Comment:
Good points. I was being a bit too trusting I think. I wrote a script to check the validity of the files in the data repo, but I'm not running it here. I think the best thing would be to run that check in the import command so that we can be confident in the data in the DB. Thoughts?

Reply:
I like it. Looked at the check in linked above and 👍.